docs/cookbook/shop/vue-storefront.rst

```diff
@@ -0,0 +1,196 @@
+`this <https://github.com/BitBagCommerce/SyliusVueStorefront2Plugin/blob/main/doc/installation.md>`_ repository page.
+
+* Require plugin with composer.
+
+.. code-block:: bash
+
+    composer require ``bitbag/vue-storefront2-plugin``
+
+* Add plugin dependencies to your ``config/bundles.php`` file.
+
+.. code-block:: php
+
+    return [
+     ...
+
+        BitBag\SyliusVueStorefront2Plugin\BitBagSyliusVueStorefront2Plugin::class => ['all' => true],
+    ];
+
+* Enable API in ``config/services.yaml``.
+
+.. code-block:: yaml
+
+    sylius_api:
+        enabled: true
+
+* Add plugin mapping path to your ``config/packages/api_platform.yaml`` file as the last element.
+
+.. code-block:: yaml
+
+    api_platform:
+        mapping:
+            paths:
+                - '%kernel.project_dir%/vendor/bitbag/vue-storefront2-plugin/src/Resources/api_resources'
+
+* Import serialization files in ``config/packages/framework.yaml`` file.
+
+.. code-block:: yaml
+
+    framework:
+        serializer:
+            mapping:
+                paths:
+                    - '%kernel.project_dir%/vendor/bitbag/vue-storefront2-plugin/src/Resources/serialization'
+
+* Import plugin config files in ``config/packages/bitbag_sylius_vue_storefront2_plugin.yaml``.
+
+.. code-block:: yaml
+
+    imports:
+        - { resource: "@BitBagSyliusVueStorefront2Plugin/Resources/config/services.xml" }
+
+You are free to adjust two parameters in the same file:
+
+.. code-block:: yaml
+
+    bitbag_sylius_vue_storefront2:
+        refresh_token_lifespan: 2592000 # default value
+        test_endpoint: 'http://127.0.0.1:8080/api/v2/graphql' # default value
+
+* Add some external Doctrine mappings.
+
+.. code-block:: yaml
+
+    doctrine:
+        orm:
+            mappings:
+                VueStorefront2:
+                    is_bundle: false
+                    type: xml
+                    dir: '%kernel.project_dir%/vendor/bitbag/vue-storefront2-plugin/src/Resources/doctrine/model'
+                    prefix: 'BitBag\SyliusVueStorefront2Plugin\Model'
+                    alias: BitBag\SyliusVueStorefront2Plugin
+
+* Change the Sylius Taxon repository class to add some queries required by GraphQL in ``config/packages/_sylius.yaml``.
+
+.. code-block:: yaml
+
+    sylius_taxonomy:
+       resources:
+          taxon:
+             classes:
+                repository: BitBag\SyliusVueStorefront2Plugin\Doctrine\Repository\TaxonRepository
+
+* If you're extending Sylius `ProductAttributeValue` entity, please use our trait inside: `BitBag\SyliusVueStorefront2Plugin\Model\ProductAttributeValueTrait`. Otherwise, please create an entity, which uses the trait and setup the Sylius resource in ``config/packages/_sylius.yaml``. Read more on how to customize models in a `different part of Sylius docs <https://sylius-older.readthedocs.io/en/latest/customization/model.html>`_ if you are not familiar with the process yet.
```

Comment:
```suggestion
* If you're extending Sylius `ProductAttributeValue` entity:

Please use our trait inside: ``BitBag\SyliusVueStorefront2Plugin\Model\ProductAttributeValueTrait``. 
Otherwise, please create an entity, which uses the trait and setup the Sylius resource in ``config/packages/_sylius.yaml``. 

Read more on how to customize models in a `different part of Sylius docs <https://sylius-older.readthedocs.io/en/latest/customization/model.html>`_ 
if you are not familiar with the process yet.
```